content/en/dynamic_instrumentation/_index.md

```diff
@@ -114,27 +111,54 @@ For more information, read [How Dynamic Instrumentation Works][11].
 
 To create a metric probe:
 
-1. Complete the generic probe setup (choose service, environment, version, and probe location).
 1. Select **Metric** as the probe type.
+1. Complete the generic probe setup (choose service, environment, version, and probe location).
 1. Specify a name for the metric, which will be prefixed with `dynamic.instrumentation.metric.probe.`.
 1. Select a metric type (count, gauge, or histogram).
-1. Choose the value of the metric using the Debugger expression language. For count metrics this is optional, and if you omit it, every invocation increments the count by one.
+1. Choose the value of the metric using the [Dynamic Instrumenation expression language][15]. For count metrics this is optional, and if you omit it, every invocation increments the count by one.
 
 {{< img src="dynamic_instrumentation/metric_probe.png" alt="Creating a Dynamic Instrumentation metric probe" >}}
 
+### Creating a span probe
+
+A *span probe* emits a span when a method is executed. You can use a *span probe* as a more efficient alternative to [creating new spans with Custom Instrumentation]][13], as it does not require making code changes and redeploying your software. If the method throws an exception, the details of the exception will be associated with the newly created span's `error` tag.
+
+To create a span probe:
+
+1. Select **Span** as the probe type.
+1. Complete the generic probe setup (choose service, environment, version, and probe location). 
+
+
+### Creating a span tag probe
+
+A *span tag* probe decorates an existing span with a tag value.  You can use a *span tag probe* as a more efficient alternative to [using Custom Instrumentation to addings tags in code][14], as it does not require making code changes and redeploying your software.
+
+To create a span probe:
```

Comment:
To create a span tag probe

Reply:
Fixed in 0e2f1be7a5d7234bc8981a32d5bd14c76c9b18f4